internal/workflow/processing_test.go

```diff
@@ -71,8 +71,11 @@ func (s *ProcessingWorkflowTestSuite) SetupWorkflowTest(useAm bool) {
 	s.env.RegisterActivityWithOptions(am.NewPollIngestActivity(
 		logger, &am.Config{}, amclienttest.NewMockIngestService(ctrl),
 		dur).Execute, temporalsdk_activity.RegisterOptions{Name: am.PollIngestActivityName})
-
-	s.workflow = NewProcessingWorkflow(logger, pkgsvc, wsvc, useAm)
+	if useAm {
```

Comment:
I think we should pass the taskqueue name directly to this method, e.g. 

```
func (s *ProcessingWorkflowTestSuite) SetupWorkflowTest(presTaskQueue string) {
```

Then this conditional block can be reduced to:
```
s.workflow = NewProcessingWorkflow(logger, pkgsvc, wsvc, presTaskQueue)
```